app/recover-password/page.tsx

```diff
@@ -0,0 +1,91 @@
+"use client";
+import React, { useState } from "react";
+import axios from "axios";
+import { TextField, Button, Typography, Container } from "@mui/material";
+
+const ForgotPassword: React.FC = () => {
```

Comment:
Wouldn't it be better to create a React component for this rather than a new page?